webpack/assets/javascripts/react_app/components/notifications/Notification.js

```diff
@@ -18,6 +18,33 @@ const Notification = ({created_at, seen, text, level, id}) => {
     (<strong {...tooltip} className="drawer-pf-notification-message pointer"
              onClick={markAsRead}>{text}</strong>);
 
+  const links = [];
+
+  let actions_dropdown = '';
+
```

Comment:
This component may best be put on it's own as a seperate component, accepting a list of links, and then on final render
{ actions.links && <ActionsDropdown links={ actions.links }  /> }
that way we dont inflate the notification component too much, and dont have to deal with different types of notifications